### ISSUE TITLE
Chapter

### ISSUE THREAD

**Issue by mulundapm** [bcc-code/bcc-media-platform#474]

**Comment by andreasgangso** [comment]
It seems common for chapters to only have a start time.
At least for display, it makes it easier to only have one point to deal with, otherwise you could end up with some weird sections.

Youtube shows the chapter as a block of time:

![Image](https://github.com/bcc-code/bcc-media-platform/assets/727125/45640813-a5bc-4e98-84a2-9dffc5071b44)


NRK calls them "index points" and shows them as points in time instead of a block of time (chapter).
Here hovering a chapter point:

![Image](https://github.com/bcc-code/bcc-media-platform/assets/727125/b14b3941-2678-4f73-8654-c00e372e7f92)

And hovering between the points just shows the time

![Image](https://github.com/bcc-code/bcc-media-platform/assets/727125/498edd8f-6a71-4ac7-81d5-9490d2d4e847)

**Comment by mulundapm** [comment]
Was thinking to have in and out point because not everything within an episode should be marked within a block of time. In that case, like the idea of index point, then ui wise not every part of the content have to be a within a block of time, where we could only mark the important starting point.

**Comment by andreasgangso** [comment]
Done in v4.6.8

**Comment by mulundapm** [comment]
Does it include analytics?

Chapter clicked?

**Comment by andreasgangso** [comment]
with elementType, elementId, chapterStart?

**Comment by mulundapm** [comment]
elementType, elementId, chapterTitle, chapterType, chapterStart

**Comment by andreasgangso** [comment]
@fredrikved I need chapter type in graphql, see above

**Comment by fredrikved** [comment]
Chapter ID should be exposed. Do you need the rest then? @mulundapm

**Comment by andreasgangso** [comment]
Yeah its a bit simpler for analytics if I pass through the data than syncing the data to bigquery and joining it in

**Comment by mulundapm** [comment]
Whats contained in chapter ID? And how would this be included in metabase? @fredrikved

**Comment by fredrikved** [comment]
Chapter ID is just a unique ID for the chapter. 
The rest would have to be joined in from another table or something in metabase whenever you need it.
